frontend/src/bundles/common/components/video-modal/components/video-modal-content/helpers/sanitize-json-string/sanitize-json-string.ts

```diff
@@ -0,0 +1,7 @@
+const sanitizeJsonString = (input: string): string => {
+    let sanitized = input.replaceAll('`', '');
+    sanitized = sanitized.replaceAll('json', '');
```

Comment:
```suggestion
    const sanitized = input.replaceAll('`', '').replaceAll('json', '');
```
I guess you can chain `replaceAll`